matter_server/client/client.py

```diff
@@ -125,7 +121,8 @@ def get_node(self, node_id: int) -> MatterNode:
         """Return Matter node by id or None if no node exists by that id."""
         if node := self._nodes.get(node_id):
             return node
-        raise NodeNotExists(f"Node {node_id} does not exist or is not yet interviewed")
+        msg = f"Node {node_id} does not exist or is not yet interviewed"
+        raise NodeNotExists(msg)
```

Comment:
I don't understand what makes it less readable in the traceback:
https://docs.astral.sh/ruff/rules/f-string-in-exception/

Not sure about this one.

Reply:
yeah I was surprised by it as well - we can disable this one if you want

Reply:
@frenck @MartinHjelmare do we want this rule to be disabled by default ?

Reply:
Personally I don't think this rule improves things.

Reply:
Me too, I'm in favor as well to just disable it